packages/mongo/mongo_livedata_tests.js

```diff
@@ -1510,7 +1520,8 @@ testAsyncMulti('mongo-livedata - transform sets _id if not present, ' + idGenera
   function (test, expect) {
     var self = this;
     var justId = function (doc) {
-      return _.omit(doc, '_id');
+      const {_id, ...rest} = doc;
```

Comment:
It'd be better not to create such unused variables. Instead, you could do a shallow copy and `delete` it.